src/lib/template-result.ts

```diff
@@ -68,8 +74,19 @@ export class TemplateResult {
   }
 
   getTemplateElement(): HTMLTemplateElement {
-    const template = document.createElement('template');
-    template.innerHTML = this.getHTML();
+    let template;
+    if (needTemplate) {
+      template = contentDoc.createElement('div') as any;
+      template.content = contentDoc.createDocumentFragment();
+      const body = contentDoc.body;
+      body.innerHTML = this.getHTML();
+      while (body.firstChild) {
```

Comment:
we have a reparentNodes function here: https://github.com/Polymer/lit-html/blob/master/src/lib/dom.ts#L24

Reply:
done